src/entity/Settings.ts

```diff
@@ -19,6 +19,21 @@ export enum ChecklistViewState {
   Hidden = 'hidden',
 }
 
+export type SettingsFlags = Partial<{
```

Comment:
This is usually enum and then we can reuse it with toGqlEnum for schema definition.

Reply:
@capJavert I don't this this is true, can't find an example of enum base flags?

Reply:
Ignore this 😆 I was looking at completely wrong place, you can't really make flags from enum either way.